src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -180,6 +185,19 @@ public boolean configure(StaplerRequest staplerRequest, JSONObject json) throws
     {
       activeIndexer = logstashIndexer;
     }
+    
+    try {
```

Comment:
This check need to be done before assigning it to activeIndexer as activeIndexer is the one that is really used.